cita-network/src/p2p_protocol/transfer.rs

```diff
@@ -114,5 +118,7 @@ pub fn create_transfer_meta(
             });
             ProtocolHandle::Callback(handle)
         })
+        .name(|_| "/cita/transfer".to_owned())
+        .support_versions(vec!["0.0.1".to_owned()])
```

Comment:
CITA v0.22.0  supported `0.0.1`,  But this version will incompatible with v0.22.0. So this `support_versions` should not include `0.0.1`, this can be `0.0.2`.

Reply:
if you have rule ,declare it .
I think this number make no sense,now

Reply:
p2p protocol will check for its support versions while transferring data.
The codec of the discovery protocol for this version is different from v0.22.0, so they can not discovery each other in the same network.